sync-main-transifex

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+git checkout master
+git checkout -b sync-main-transifex
+
+./autogen.sh --prefix=/usr
+
+rm po/*.po
+tx pull -r MATE.master--mate-utils --minimum-perc 10 --no-interactive -a -f
```

Comment:
`minimum_perc = 2` is already set in .tx/config.
Why overwriting existing setting which we use in all repos?